tests_v2/test_bell_tab_indicator.py

```
@@ -0,0 +1,171 @@
+
+def wait_for_bell(client: cmux, surface_id: str, present: bool, timeout: float = 2.0) -> bool:
+    deadline = time.time() + timeout
+    while time.time() < deadline:
+        bells = client.list_bell_surfaces()
+        if (surface_id in bells) == present:
+            return True
+        time.sleep(0.05)
+    return False
+
+
+def main() -> int:
```

Ruff (0.15.12)

[warning] Too many return statements (14 > 6)

(PLR0911)